mujoco_warp/_src/constraint.py

```diff
@@ -221,7 +226,9 @@ def _efc_equality_connect(
       0.0,
       Jqvel[i],
       0.0,
+      int(ConstraintType.EQUALITY.value),
```

Comment:
nit: I think you can drop the `.value`

(comment applies here and elsewhere)

Reply:
it seems like `int(ConstraintType.EQUALITY)` does not work, but `ConstraintType.EQUALITY.value` works. should i update relevant cases in `constraint.py` to the latter format?

Reply:
Sure that works.